src/posix/platform/system.cpp

```diff
@@ -127,26 +130,11 @@ void otSysSetInfraNetif(const char *aInfraNetifName, int aIcmp6Socket)
 }
 #endif
 
-void platformInit(otPlatformConfig *aPlatformConfig)
+void _platformInitCoreMode(otPlatformConfig *aPlatformConfig)
```

Comment:
Do we have the code style to add the `_` before the function? Could we add `static` before the function?

Reply:
I've removed `_`. It makes sense to add static. However currently none of the `platformXXX` has `static` prepended. So I think let's keep the consistency.